server/src/main/kotlin/com/hockey/elo/elotracker/match/model/MatchRecord.kt

```diff
@@ -8,12 +8,10 @@ import javax.persistence.Id
 @Entity
 data class MatchRecord(
     @Id @GeneratedValue
-    var id: Long,
-    var playerOneId: Long,
-    var playerTwoId: Long,
+    var id: Long = 0L,
```

Comment:
Do we need to have a default set here since it is generated by the DB?

Reply:
I will test and find out.

Reply:
if you want to make a new instance without having to define all of the parameters you do need it to be defaulted or optional. The DB is going to override it anyways so I would leave this.

Reply:
Ok, I legit didn't know. Kotlin is new and scary and fun